src/collectd.conf.in

```diff
@@ -1402,6 +1402,7 @@
 #       Version 2
 #       Community "another_string"
 #       Collect "std_traffic" "hr_users"
+#       Bulk_Size 100
```

Comment:
`BulkSize 0`

Reply:
I was under the impression that the point of thoses examples was to provide samples of non default settings.
Maybe should i a new example for "highend.switch.mydomain.org" ?

Reply:
you can leave 100, but please fix the Keyword

Reply:
Yes that i did anyway !